### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by miguelbrandao**

### CODE REVIEW

File: .github/workflows/ci-style.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    branches: [main, develop]
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: ["16.13.2"]
```
Comment (nelsonmestevao) [inline .github/workflows/ci-style.yml:14]:
```suggestion
        node-version: [16.x]
```

Comment (miguelbrandao) [reply .github/workflows/ci-style.yml:14]: But the project is locked to 16.13.2 or higher

Comment (nelsonmestevao) [reply .github/workflows/ci-style.yml:14]: It's suppose to work will all nodejs versions 16 forward, don't you agree? I think it's easier to keep this updated if we don't pin the small versions on the CI.

**Review by nelsonmestevao — APPROVED** [review]
:runner:

**Pull request merged**